src/consensus/pbft/pbft_types.h

```diff
@@ -198,6 +201,31 @@ namespace pbft
       return kv::NoVersion;
     }
 
+    kv::TxID next_txid()
+    {
+      while (true)
```

Comment:
These still have the `while (true)` loop, which has been removed from the other methods.

Reply:
Sorry, thanks for spotting that.